Frontend/src/Components/Shared/RangeRow.js

```diff
@@ -0,0 +1,65 @@
+import {Flex, Center, Text, Spacer, useColorMode} from "@chakra-ui/react";
+import getColor from "../Shared/colors";
+import RangeBar from "./RangeBar";
+
+export default function RangeRow(props) {
+    const { colorMode } = useColorMode();
+    const borderColor = getColor("border", colorMode);
+
+    const bg = (props.dataValue < props.dataConstant.MIN || props.dataValue > props.dataConstant.MAX) ?
+        getColor("errorBg", colorMode) : props.bg;
+
+    // set the decimal point to 3 by default
+    const decPoint = (props.DecimalPoint !== undefined) ? props.DecimalPoint : 3;
+    
+    let lineHeight;
+    let textSize;
+    let rangeBarHeight;
+    if (props.size === 'sm' || props.size === undefined) {
```

Comment:
```suggestion
    if ((props.size ?? 'sm') === 'sm') {
```
Again, mostly for consistency, let's use the shorthand for handling undefined variables